### PR TITLE
LPS-199879 Remove metal-uri from osb-faro-web

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+
+function isDef(param) {
+	return param !== null && param !== undefined;
+}
+
+function addParam(url, key, value) {
+	url.searchParams.delete(key, value);
+
+	if (isDef(key) && isDef(value)) {
+		url.searchParams.append(key, value);
```
Comment (victorg1991) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts:24]: can we use `set` instead of delete + append?

**Comment by victorg1991** [comment]
Hey @sandrodw3  Just a small thing, review it and sent to the team :P

**Comment by sandrodw3** [comment]
https://github.com/liferay-ac/liferay-portal/pull/957

**Pull request closed without merging**